src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IDelegateCreationExpression.vb

```diff
@@ -0,0 +1,966 @@
+              Conversion: CommonConversion (Exists: True, IsIdentity: False, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+              Operand: ILiteralExpression (OperationKind.LiteralExpression, Type: System.Int32, Constant: 1, IsInvalid) (Syntax: '1')
+        ILabeledStatement (Label: exit) (OperationKind.LabeledStatement, IsInvalid) (Syntax: 'Function() 1')
+          Statement: null
+        IReturnStatement (OperationKind.ReturnStatement, IsInvalid) (Syntax: 'Function() 1')
+          ReturnedValue: ILocalReferenceExpression:  (OperationKind.LocalReferenceExpression, Type: System.String, IsInvalid) (Syntax: 'Function() 1')
+]]>.Value
+
+            Dim expectedDiagnostics = <![CDATA[
+BC30512: Option Strict On disallows implicit conversions from 'Integer' to 'String'.
+        Dim a As Func(Of String) = CType(Function() 1, Func(Of String))'BIND:"CType(Function() 1, Func(Of String))"
+                                                    ~
+]]>.Value
+
+            VerifyOperationTreeAndDiagnosticsForTest(Of CTypeExpressionSyntax)(source, expectedOperationTree, expectedDiagnostics)
+        End Sub
+
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact()>
+        Public Sub DelegateCreationExpression_ExplicitLambdaConversion_ReturnRelaxation()
+            Dim source = <![CDATA[
+Option Strict On
+Imports System
+Module M1
+    Sub Main()
+        Dim a As Func(Of Object) = CType(Function() 1, Func(Of Object))'BIND:"CType(Function() 1, Func(Of Object))"
+    End Sub
+End Module]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IDelegateCreationExpression (OperationKind.DelegateCreationExpression, Type: System.Func(Of System.Object)) (Syntax: 'CType(Funct ... Of Object))')
+  Target: IAnonymousFunctionExpression (Symbol: Function () As System.Object) (OperationKind.AnonymousFunctionExpression, Type: null) (Syntax: 'Function() 1')
+      IBlockStatement (3 statements, 1 locals) (OperationKind.BlockStatement) (Syntax: 'Function() 1')
+        Locals: Local_1: <anonymous local> As System.Object
+        IReturnStatement (OperationKind.ReturnStatement) (Syntax: '1')
+          ReturnedValue: IConversionExpression (Implicit, TryCast: False, Unchecked) (OperationKind.ConversionExpression, Type: System.Object) (Syntax: '1')
+              Conversion: CommonConversion (Exists: True, IsIdentity: False, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+              Operand: ILiteralExpression (OperationKind.LiteralExpression, Type: System.Int32, Constant: 1) (Syntax: '1')
+        ILabeledStatement (Label: exit) (OperationKind.LabeledStatement) (Syntax: 'Function() 1')
+          Statement: null
+        IReturnStatement (OperationKind.ReturnStatement) (Syntax: 'Function() 1')
+          ReturnedValue: ILocalReferenceExpression:  (OperationKind.LocalReferenceExpression, Type: System.Object) (Syntax: 'Function() 1')
```

Comment:
>ReturnedValue: ILocalReferenceExpression:  (OperationKind.LocalReferenceExpression, Type: System.Object) (Syntax: 'Function() 1') [](start = 10, length = 129)

It is not clear what local does this node reference.

Reply:
This is apparently a SynthesizedLocal (of type FunctionReturnValue) introduced by the LambdaBodyBinder.

---
In reply to: [138746756](https://github.com/dotnet/roslyn/pull/22056#discussion_r138746756) [](ancestors = 138746756)

Reply:
Let's open an issue to investigate whether we can/should remove these compiler generated artifacts like this extra return, the exit label, etc 

---
In reply to: [141468218](https://github.com/dotnet/roslyn/pull/22056#discussion_r141468218) [](ancestors = 141468218,138746756)

Reply:
This is probably an extension on [https://github.com/dotnet/roslyn/issues/7397](https://github.com/dotnet/roslyn/issues/7397).

---
In reply to: [141977566](https://github.com/dotnet/roslyn/pull/22056#discussion_r141977566) [](ancestors = 141977566,141468218,138746756)